### PR TITLE
Replaced subjects

### PR DESCRIPTION
**Pull request by leohemsted**
In the two new tickets to add the final compiled message body to the `POST /notifications/_ _ _ _`  and  `GET /notifications[/ _ _ _ _]` responses, we neglected to convert email subjects with placeholders in them

Tests and code for that.

### CODE REVIEW

File: app/schemas.py [hunk 1]
```diff
@@ -267,10 +267,13 @@ def handle_personalisation_property(self, in_data):
     def handle_template_merge(self, in_data):
         if in_data.get('personalisation'):
             from notifications_utils.template import Template
-            merged = Template(in_data['template'], in_data['personalisation']).replaced
-            in_data['body'] = merged
+            merged = Template(in_data['template'], in_data['personalisation'])
+            in_data['body'] = merged.replaced
```
Comment (quis) [inline app/schemas.py]: Why not
```
in_data['body'] = Template(in_data['template'], in_data['personalisation']).replaced
```
?

Comment (quis) [inline app/schemas.py]: One less ambiguously named variable ✊

Comment (leohemsted) [inline app/schemas.py]: i use it on the next line - so need to save it rather than the previous one-liner. happy to rename it to `template` tho

**Comment by quis** [comment]
👍

**Pull request merged**